CRM/Report/Form/Contribute/Summary.php

```diff
@@ -41,7 +41,7 @@ class CRM_Report_Form_Contribute_Summary extends CRM_Report_Form {
   protected $_customGroupExtends = array('Contribution', 'Contact', 'Individual');
   protected $_customGroupGroupBy = TRUE;
 
-  public $_drilldownReport = array('contribute/detail' => 'Link to Detail Report');
```

Comment:
I think this should be in a separate PR as it is a behaviour change & we need to check if it will get buy in (I tend to agree with it). The other changes in this are just technical so better to put on 2 tracks

Reply:
@yashodha this line is the only thing that I think is blocking merge here - because I think it;s a separate change -
 although I think it would be better if we changed joinAddressFromClause to joinAddressFromContactClause - and the same on the other functions. (that would make it more in line with extendedReport and differentiate between joinAddressFromLocation from joinAddressFromContact.